packages/client-utils/src/types.d.ts

```diff
@@ -34,4 +38,12 @@ export type TypedPublished<T extends string> = T extends keyof PublishedTypeMap
           ? OutcomeRecord
           : T extends `vaultFactory.managers.manager${number}.metrics`
             ? VaultManagerMetrics
-            : unknown;
+            : T extends 'agoricNames.instance'
+              ? Array<[string, Instance]>
+              : T extends 'agoricNames.brand'
+                ? Array<[string, Brand]>
+                : T extends 'fastUsdc'
+                  ? ContractRecord
+                  : T extends 'fastUsdc.poolMetrics'
+                    ? PoolMetrics
```

Comment:
It still feels strange for client-utils to depend on fast-usdc. Should it depend on dapp-offer-up and dapp-agoric-basics likewise?

It seems like fast-usdc should extend the type from client-utils.

Reply:
It is weird but ok for now. We'll have to solve it for https://github.com/Agoric/agoric-sdk/issues/10811